sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -14003,70 +14100,67 @@
               "shortcut": ""
             }
           ],
+          "dataType": "",
+          "persistent": true,
+          "protected": false,
+          "dbIndex": false,
+          "encrypted": false,
+          "redrawOn": "",
+          "clearOnHide": true,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "allowCalculateOverride": false,
           "validate": {
             "required": true,
             "onlyAvailableItems": false,
+            "customMessage": "",
             "custom": "",
             "customPrivate": false,
+            "json": "",
             "strictDateValidation": false,
             "multiple": false,
             "unique": false
           },
-          "key": "willYouHaveAChildAge11YearsOrUnderInDaycare",
-          "customConditional": "show = false;\r\nlet  flag = \"false\";\r\nif (data.hasOwnProperty('dependants') && data.hasOwnProperty('hasDependents')){\r\n  for (let i = 0; i < data.dependants.length; i++) {\r\n    if ((+data.dependants[i].validDependent >= 0) && (+data.dependants[i].validDependent <= 11)){\r\n      flag =\"true\";\r\n    }\r\n  }\r\n  show =  data.hasDependents === \"yes\" &&  flag === \"true\";\r\n}",
-          "type": "radio",
-          "input": true,
-          "placeholder": "",
-          "prefix": "",
-          "customClass": "",
-          "suffix": "",
-          "multiple": false,
-          "defaultValue": null,
-          "protected": false,
-          "unique": false,
-          "persistent": true,
-          "hidden": false,
-          "clearOnHide": true,
-          "refreshOn": "",
-          "redrawOn": "",
-          "modalEdit": false,
-          "dataGridLabel": false,
-          "labelPosition": "top",
-          "description": "",
           "errorLabel": "",
-          "tooltip": "",
-          "hideLabel": false,
-          "tabindex": "",
-          "disabled": false,
-          "autofocus": false,
-          "dbIndex": false,
-          "customDefaultValue": "",
-          "calculateValue": "",
-          "calculateServer": false,
-          "widget": null,
-          "attributes": {},
-          "validateOn": "change",
+          "key": "haveDaycareCosts11YearsOrUnder",

```

Comment:
This was a change I did so it is part of your form definition upload. I will not upload a separate definition